src/main/java/foodwhere/storage/JsonSerializableAddressBook.java

```diff
@@ -14,7 +14,7 @@
 import foodwhere.model.stall.Stall;
 
 /**
- * An Immutable AddressBook that is serializable to JSON format.
+ * An Immutable AddressBook in FoodWhere that is serializable to JSON format.
```

Comment:
Perhaps Immutable should be uncapitalized "immutable"?

Reply:
Resolved in latest commit